getting-started/blazor/your-first-chart-using-blazor.md

```diff
@@ -0,0 +1,193 @@
+---
+title: Your First Chart in Blazor using FusionCharts | FusionCharts
+description: This article outlines the steps to be executed for creating your first chart using the blazor-fusioncharts component.
+heading: Create a Chart in Blazor using FusionCharts
+---
+
+FusionCharts is a JavaScript charting library that enables you to create interactive charts, gauges, maps, and dashboards in JavaScript. The `blazor-fusioncharts` component lets you easily elevate data visualization to any Blazor project. On this page, we'll see how to install FusionCharts and render a chart using the `blazor-fusioncharts` component.
+
+ Integrating FusionCharts into Blazor applications can help users create visually appealing and informative data visualizations, enhancing the overall experience. 
+
+## Prerequisite
+
+Please ensure you have Visual Studio (Used Community Edition 2022 - 17.6.1) installed or updated as a prerequisite. 
+
+**IMPORTANT NOTE:** Install Blazor if not present already.
+
+
+## Installation 
+
+Follow these simple steps to install Blazor.
+
+1. Clone the code repository to your local machine from [blazor-fusioncharts](https://github.com/fusioncharts/blazor-fusioncharts).
+
+**For Windows**: 
+1. Ensure the following workloads are selected during the Visual Studio installation:
+- ASP.NET and Web Development
+- Azure Development
+- .NET Desktop
+
+2. After installation, open the Visual Studio application and select the “Open a project solution” option. 
+
+3. Navigate and select the ‘BlazorApp1.sln’ file in the extracted folder. 
+
+4. Finally, run the ‘BlazorApp1.sln’ file by clicking the run button in the toolbar. 
+
+**For MacOS and Linux**:
+1. Open the terminal from your project directory and run the ‘dotnet watch’ command.
+
+2. Open a web browser and navigate to the appropriate address to view and interact with your Blazor application.  
+
+**Setup using NuGet Package**:
+1. Open the project inside the Visual Studio app.
+
+2. On the Settings icon, select “Options”.
+
+3. Find the NuGet Package Manager, and select Package Sources from the left panel of the Options window.
+
+4. To add a package, click the “+” symbol.
+- For the Name field, add the name of the package source as ‘myget.org’.
+- For the Source field, paste the myGet Package link.
+- Click “Update”, and then click “OK”.
+
+5. Under Tools, click the “NuGet Package Manager” option and select Manage NuGet Packages for Solution.   
+
+6. Next, change the project source to “myget.org” in the top-right corner. 
```

Comment:
why are we referring to myget? was this in the readme?

Reply:
yes, this is step 7 in the NuGet Packet instructions of the README file.
Should I change this or keep it?